### PR TITLE
cleanup: correct typo in travis scripts

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: scripts/travis-functest.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -e
 
 # This script will be used by travis to run functional test
-# against different kuberentes version
+# against different kubernetes version
```
Comment (Madhu-1) [inline scripts/travis-functest.sh]:
```suggestion
# against different Kubernetes version
```

File: scripts/travis-helmtest.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 set -e
 
 # This script will be used by travis to run functional test
-# against different kuberentes version
+# against different kubernetes version
```
Comment (Madhu-1) [inline scripts/travis-helmtest.sh]:
```suggestion
# against different Kubernetes version
```

**Review by Madhu-1 — CHANGES_REQUESTED** [review]

**Comment by humblec** [comment]
@Madhu-1 addressed the comment, ptal.  Thanks

**Review by Madhu-1 — APPROVED** [review]

**Review by Yuggupta27 — APPROVED** [review]

**Comment by Madhu-1** [comment]
@mergifyio backport release-v3.3

**Pull request merged**